test/test_wkhtmltopdf_binary.rb

```diff
@@ -0,0 +1,49 @@
+require 'minitest/autorun'
+require 'wkhtmltopdf_binary'
+
+class WkhtmltopdfBinaryTest < Minitest::Test
+  def test_with_known_arch_and_os
```

Comment:
not enough tests: we should cover the case that checks the whole path to the file, it's very crucial.